src/pages/deals.html

```diff
@@ -0,0 +1,78 @@
+<header></header>
+
+<container>
+  <dealshero
+    title="Chris, is it snowing yet?"
+    text="The temperature inside this apple pie is over one thousand degrees. If I squeeze it, a jet of molten bramey apple will squirt out. Could go your way; could go mine. Either way, one of us is going down."
+    top-image="http://placehold.it/580x400"
+    bottom-image="http://placehold.it/580x75"
+    href="#link"
+  ></dealshero>
+  
+  <row>
+    <columns small="12" large="12">
+      <largedealcard
+        image="http://placehold.it/640x320"
+        title="Up to 70% off outlet items"
+        company="Amazon"
+        button-text="Button Label"
+        button-href="#link"
+      ></largedealcard>
+      
+      <largedealcard
+        image="http://placehold.it/640x320"
+        title="Up to 70% off outlet items"
+        company="Amazon"
+        button-text="Button Label"
+        button-href="#link"
+      ></largedealcard>
+      
+      <smalldealcard
+        image="http://placehold.it/200x200"
+        title="Up to 70% off outlet items"
+        company="Amazon"
+        button-text="Button Label"
+        button-href="#link"
+      ></smalldealcard>
+      
+      <smalldealcard
+        image="http://placehold.it/200x200"
+        title="Up to 70% off outlet items"
+        company="Amazon"
+        button-text="Button Label"
+        button-href="#link"
+      ></smalldealcard>
+      
+      <row>
+        <columns small="12" large="6">
+          <spacer size="30"></spacer>
+          <inlinedealcard
+            image="http://placehold.it/640x320"
+            title="Up to 70% off outlet items"
+            company="Amazon"
+            button-text="Button Label"
+            button-href="#link"
+          ></inlinedealcard>
+        </columns>
+        <columns small="12" large="6">
+          <spacer size="30"></spacer>
+          <inlinedealcard
+            image="http://placehold.it/640x320"
+            title="Up to 70% off outlet items"
+            company="Amazon"
+            button-text="Button Label"
+            button-href="#link"
+          ></inlinedealcard>
+        </columns>
+      </row>
+      
```

Comment:
@adtaylor The above block translates to:

![screen shot 2016-06-01 at 14 45 43](https://cloud.githubusercontent.com/assets/13519563/15711802/93f65f18-2807-11e6-8f67-8113e471bfd0.png)

Do you think that should be componentised once more into a `inlineDealCardRow`? You'd have to provide two of each of the parameters which is a bit awkward, but it would completely obscure the details of the rows/columns from the user.